### PR TITLE
Fixed header button

### LINKED ISSUE #13: fix header buttons
Unlink the header buttons from colorlib and Redirect them on the same page 

image

### PR DESCRIPTION
**Pull request by Yashs911**
Fixes #13

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -36,41 +36,41 @@
 <div class="row align-items-center">
 <div class="col-lg-12">
 <nav class="navbar navbar-expand-lg navbar-light">
-<a class="navbar-brand" href="https://colorlib.com/preview/theme/amor/index.html"> <img src="images/logo.png" alt="logo"> </a>
+<a class="navbar-brand" href="index.html"> <img src="images/logo.png" alt="logo"> </a>
```
Comment (Dishebh) [inline index.html:39]:
```suggestion
<a class="navbar-brand" href="/"> <img src="images/logo.png" alt="logo"> </a>

```

**Review by Dishebh — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Yashs911) [reply index.html]: Done

**Comment by Dishebh** [comment]
Please make the same changes in the other links (that you've updated) as well :)

File: index.html [hunk 2]
```diff
@@ -36,41 +36,41 @@
...
-<a class="navbar-brand" href="https://colorlib.com/preview/theme/amor/index.html"> <img src="images/logo.png" alt="logo"> </a>
+<a class="navbar-brand" href="/"> <img src="images/logo.png" alt="logo"> </a>
 <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
 <span class="ti-menu"></span>
 </button>
 <div class="collapse navbar-collapse main-menu-item justify-content-end" id="navbarSupportedContent">
 <ul class="navbar-nav align-items-center">
 <li class="nav-item">
-<a class="nav-link" href="https://colorlib.com/preview/theme/amor/index.html">Home</a>
+<a class="nav-link" href="index.html">Home</a>
```
Comment (Dishebh) [inline index.html:46]:
```suggestion
<a class="nav-link" href="/">Home</a>

```

File: index.html [hunk 3]
```diff
@@ -36,41 +36,41 @@
...
 </li>
 <li class="nav-item">
-<a class="nav-link" href="https://colorlib.com/preview/theme/amor/about.html">about</a>
+<a class="nav-link" href="about.html">about</a>
```
Comment (Dishebh) [inline index.html:49]:
```suggestion
<a class="nav-link" href="/">about</a>

```

File: index.html [hunk 4]
```diff
@@ -36,41 +36,41 @@
...
 </li>
 <li class="nav-item">
-<a class="nav-link" href="https://colorlib.com/preview/theme/amor/Causes.html">Causes</a>
+<a class="nav-link" href="Causes.html">Causes</a>
```
Comment (Dishebh) [inline index.html:52]:
```suggestion
<a class="nav-link" href="/">Causes</a>

```

File: index.html [hunk 5]
```diff
@@ -36,41 +36,41 @@
...
 </li>
 <li class="nav-item dropdown">
-<a class="nav-link dropdown-toggle" href="https://colorlib.com/preview/theme/amor/blog.html" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+<a class="nav-link dropdown-toggle" href="blog.html" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
```
Comment (Dishebh) [inline index.html:55]:
```suggestion
<a class="nav-link dropdown-toggle" href="/" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">

```

File: index.html [hunk 6]
```diff
@@ -623,7 +623,7 @@ <h4>First cattle which earthcan get
 <img src="images/blog_3.jpg" alt="">
 </div>
 <div class="single_appartment_content">
-<a href="https://colorlib.com/preview/theme/amor/blog.html">
+<a href="blog.html">
```
Comment (Dishebh) [inline index.html:626]:
```suggestion
<a href="/">

```

**Review by Dishebh — CHANGES_REQUESTED** [review]
Do similar changes in all others :)

**Review by Dishebh — APPROVED** [review]

**Pull request merged**